pubspec.lock

```diff
@@ -1,385 +1,174 @@
 # Generated by pub
 # See https://dart.dev/tools/pub/glossary#lockfile
 packages:
-  _fe_analyzer_shared:
```

Comment:
Were does changes deleted automatically or did you do it by hand?

Reply:
I've just checked out your branch and IDE automatically made changes to this file. Idk what is going on, probably some differences in versions 🤷‍♂️ We could either figure out what causes this and prevent it, include this file in .gitignore or stick with one version so we don't have changes in every commit.

Reply:
I think this is because I ran flutter pub get...I think we should commit this file...